WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -569,18 +570,18 @@ extension AztecPostViewController {
 //
 extension AztecPostViewController {
     @IBAction func publishButtonTapped(sender: UIBarButtonItem) {
-        handlePublishButtonTapped(secondaryPublishTapped: false)
+        handlePublishButtonTapped(shouldDismissEditorWhenDone: true)
```

Comment:
IMO `shouldDismissEditorWhenDone` is a bit too wordy for swift (so is `handlePublishButtonTapped` 😝). How about something like `publishTapped(dismissWhenDone:)` or `publishTapped(autoDismiss:)`?